src/04-modality-specific-files/05-task-events.md

```diff
@@ -93,29 +114,76 @@ Example:
+
+| Field name           | Description                                                                                                                                                                                                                                                        |
+|----------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| StimulusPresentation | RECOMMENDED. [Object][] containing key value pairs related to the software used to present the stimuli during the experiment, specifically: `OperatingSystem`, `SoftwareName`, `SoftwareRRID`, `SoftwareVersion` and `Code`. See table below for more information. |
+
+The object supplied for `StimulusPresentation` SHOULD include the following key-value pairs:
+
+| Key name        | Description                                                                                                                                                                                                                   |
+|-----------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| OperatingSystem | RECOMMENDED. Operating system used to run the stimuli presentation software (for formatting recommendations, see examples below this table).                                                                                  |
+| SoftwareName    | RECOMMENDED. Name of the software that was used to present the stimuli.                                                                                                                                                       |
+| SoftwareRRID    | RECOMMENDED. [Research Resource Identifier](https://scicrunch.org/resources) of the software that was used to present the stimuli. Examples: The RRID for Psychtoolbox is 'SCR_002881', and that of PsychoPy is 'SCR_006571'. |
+| SoftwareVersion | RECOMMENDED. Version of the software that was used to present the stimuli.                                                                                                                                                    |
+| Code            | RECOMMENDED. [URI][uri] of the code used to present the stimuli. Persistent identifiers such as DOIs are preferred. If multiple versions of code may be hosted at the same location, revision-specific URIs are recommended.  |
+
+The operating system description should include:
+-   type (Windows, MacOS, Linux),
+-   distribution (if applicable: Ubuntu, Debian)
+-   the version number or equivalent code name
+
+Examples:
+-   Windows - 10
+-   macOS - Catalina or macOS - 10.15
+-   Linux - Ubuntu - Bionic Beaver
+-   Linux - Ubuntu - 18.04.5 LTS
```

Comment:
I would drop the "named" option here, too

Reply:
+1 to drop named versions --> having one way to specify something is better than two ways ([python zen](https://www.python.org/dev/peps/pep-0020/#id2))